LuaRules/Configs/tactical_ai_defs.lua

```diff
@@ -1369,14 +1369,15 @@ local behaviourConfig = {
 		fightOnlyUnits = medRangeExplodables,
 		maxSwarmLeeway = 30,
 		minSwarmLeeway = 130,
-		jinkPeriod = 2,
+		jinkPeriod = 2.5,
```

Comment:
As far as I can tell, only integers make sense here and this is equivalent to 3.
```suggestion
		jinkPeriod = 3,
```